tests/test-cases/Group10-VCH-Restart/10-01-VCH-Restart.robot

```diff
@@ -145,6 +145,52 @@ Created Network And Images Persists As Well As Containers Are Discovered With Co
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull ${alpine}
     Should Be Equal As Integers  ${rc}  0
 
+
+Container on Open Network And Port Forwarding Persist After Reboot
+    [Setup]     NONE
+
+    Log To Console  Create Port Groups For Container network
+    ${out}=  Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Run  govc host.portgroup.add -vswitch vSwitchLAN open-net
+
+    Install VIC Appliance To Test Server  additional-args=--container-network=open-net --container-network-firewall=open-net:open
+
+    # Create a container on the open network
+    ${open-running}=  Launch Container  vch-restart-open-running  open-net
+    ${open-exited}=  Launch Container  vch-restart-open-exited  open-net  ls
+
+    # Create nginx on the open network and bridge network
+    Launch Container With Port Forwarding  webserver-open  10000  10001  open-net
+    Launch Container with Port Forwarding  webserver-bridge  10002  10003  bridge
+    Check Nginx Port Forwarding  10002  10003
+
+    # Reboot VCH
```

Comment:
Please use the mechanism in https://github.com/vmware/vic/pull/7133 to collect logs before rebooting the VCH here. https://github.com/vmware/vic/pull/7133 updated this test suite to collect logs every time before rebooting the VCH.